src/components/PurchaseOrder/PO.js

```diff
@@ -59,6 +64,16 @@ class PO extends Component {
         },
       },
     },
+    linesLimit: {
+      type: 'okapi',
+      records: 'configs',
+      path: 'configurations/entries',
```

Comment:
please use constant for paths, it's expected to be reused across the application